src/peripherals/pio.spec.ts

```diff
@@ -101,17 +102,27 @@ describe('PIO', () => {
     await cpu.writeUint32(SM0_INSTR, pioJMP(PIO_COND_ALWAYS, 0)); // Jump machine 0 to address 0
     // Clear FIFOs
     await cpu.writeUint32(SM0_SHIFTCTRL, FJOIN_RX);
-    await cpu.writeUint32(SM0_SHIFTCTRL, 0);
+    //values at reset
+    await cpu.writeUint32(SM0_SHIFTCTRL, (1 << 18) | (1 << 19));
```

Comment:
What are do bit 18 / 19 mean?

Let's define a constant for them (like we have `FJOIN_RX`), so the test code is easier to understand